### DISCUSSION TITLE
Center point on settings is maybe not a good UX

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#14497] [Q&A] [answered]
I am not an expert but as a user, I have a feeling like scrolling this far and still showing 80% of the settings dimmed doesn't feel right

![osu_2021-08-24_22-33-56](https://user-images.githubusercontent.com/41755816/130823594-9a509c58-e572-4b93-b9ce-b3dbead11abd.jpg)

**Comment by frenzibyte** [comment, marked as answer]
Practically a duplicate of https://github.com/ppy/osu/discussions/14414, the vertical point will get tweaked to feel better eventually.

**Reply by Gonzalo-Bruna** [reply]
Thanks, didn't see that one.
